Userland/Libraries/LibVideo/VP9/Parser.h

```diff
@@ -184,17 +192,23 @@ class Parser {
     u32 m_mi_col_end { 0 };
     u32 m_mi_row { 0 };
     u32 m_mi_col { 0 };
-    u32 m_mi_size { 0 };
+    BlockSubsize m_mi_size { 0 };
     bool m_available_u { false };
     bool m_available_l { false };
     u8 m_segment_id { 0 };
+    /// skip equal to 0 indicates that there may be some transform coefficients to read for this block; skip equal to 1
+    /// indicates that there are no transform coefficients.
+    ///
+    /// skip may be set to 0 even if transform blocks contain immediate end of block markers.
     bool m_skip { false };
```

Comment:
m_skip seems a candidate for conversion to a verbose enum

```c++
enum TransformCoefficientsPossible
{
     Yes = 0,
     No = 1,
};
```

or something? probably a follow-up, but the lengthy comment here suggests that `bool` is not the right data type for this.

Reply:
I've added a FIXME for this. Definitely another candidate for later readability refactoring.